applications/views.py

```diff
@@ -44,15 +45,23 @@ def get_queryset(self):
             return BootcampApplication.objects.prefetch_state_data()
         else:
             return (
-                BootcampApplication.objects.filter(user=self.request.user)
+                BootcampApplication.objects.prefetch_related(
+                    Prefetch(
+                        "orders",
+                        queryset=Order.objects.filter(
+                            status=Order.FULFILLED
+                        ).select_related("user__profile"),
```

Comment:
I don't think this `select_related` is necessary. I tested this without it and it worked.